starters/basic-starter/pages/[...slug].tsx

```diff
@@ -77,6 +122,12 @@ export async function getStaticProps(
 
   const type = path.jsonapi.resourceName;
 
+  let label;
+  if (path.label) {
+    // Get the taxonomy term from the path.
+    label = path.label;
```

Comment:
For consistency, we should probably get the title for taxonomy terms from the entity object like we do for nodes

Reply:
What if there is an existing taxonomy term with no associated objects like 'Software' for example

![image](https://user-images.githubusercontent.com/35243020/179066527-271bfb56-45ff-4f77-b96c-f548ece40b6a.png)

Reply:
The taxonomy term object should be still available, even if there aren't any associated entities.

Reply:
Ah yeah, I misunderstood your first comment.